src/shared/settings.js

```diff
@@ -354,6 +354,11 @@ function deepFreeze(object) {
     return Object.freeze(object);
 }
 
+// compatibility polyfill for Safari < 15.4
```

Comment:
If we are limiting browser versions at some point I recommend we add a note here as a reminder.
```suggestion
// @todo remove this polyfill when browser limitation is set above 15.4
// compatibility polyfill for Safari < 15.4
```